package.json

```diff
@@ -24,7 +26,7 @@
     "precommit": "lint-staged",
     "postcommit": "git update-index --again",
     "build": "gatsby build",
-    "develop": "gatsby develop",
+    "dev": "gatsby develop",
```

Comment:
Is there a reason to change to `dev`?

Reply:
saves having to type out `develop`.
aside from that no real convincing reasons other than that `dev` is the keyword used for a few other projects: gbptm, ARCVMarket spring to mind
your call :)

Reply:
Fair enough - but I guess we 'could' stick to the Gatsby boilerplate so it behaves like other Gatsby projects, rather than other Neontribe projects?